internal/prepare.go

```diff
@@ -148,14 +158,14 @@ func (p *Preparer) injectValues(manifestData string, dataSourceMap map[string]st
 			continue
 		}
 	}
-	return manifestData, nil
+	return manifestData
 }
 
 func generateRFC1123SubdomainCompatibleString() string {
 	rand.Seed(time.Now().UnixNano())
 	s := make([]rune, 8)
 	for i := range s {
-		s[i] = charset[rand.Intn(len(charset))]
+		s[i] = charset[rand.Intn(len(charset))] // #nosec G404: no need for crypto/rand here
```

Comment:
Can `// nolint` cover this for consistency with the other overrides?

Reply:
Done.